Lanchonete/Lanchonete/Controller/TableViewController.swift

```diff
@@ -52,4 +52,22 @@ class TableViewController: UITableViewController, TableDemoView {
         }
         return UITableViewCell()
     }
+    
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        presenter.didSelect(row: indexPath.row)
+    }
+    
+    func navigateToConfirmationScreen(sandwich: SandwichType){
+        self.performSegue(withIdentifier: "listToConfirmation", sender: sandwich)
```

Comment:
The only modification that I think could be interesting it's to move this `"listToConfirmatinon" string into a `enum` with a constant on it. You could do something like this
```Swift
enum Segues {
 static let listToConfirmation = "listToConfirmation"
}

self.performSegue(withIdentifier: Segues.listToConfirmation, sender: sandwich)
```

Reply:
The reason for doing it it's to make it easier to add more segues and also not be wrong with loose strings in the code. There will be only one place where the string should be right. The other ones will be validated during compilation time :)

Reply:
Hey Paulo, I added it, can you please take a look? Thank you.